contracts/VAIVault/IVAIVault.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+interface IVAIVault {
```

Comment:
Missing natspec for all functions in the interface?

Reply:
b1cbdadf4823e0d1007ae88db55da943a1617c4e